package.json

```diff
@@ -1,106 +1,46 @@
 {
-  "name": "@brainhubeu/react-carousel",
-  "version": "1.19.14",
-  "description": "Carousel component for React",
-  "engines": {
-    "npm": ">=6.14.5"
-  },
-  "files": [
-    "lib/*"
+  "private": true,
```

Comment:
why private?

Reply:
It prevents from accidental publishing this package hence we aim to publish only react-carousel workspace/package

Reply:
ok